windows/process.go

```diff
@@ -0,0 +1,30 @@
+// +build windows
+
+package windows
+
+import (
+	"github.com/containerd/containerd"
+	"github.com/containerd/containerd/windows/hcs"
+	"golang.org/x/net/context"
```

Comment:
Of course, I have no idea why goimport picked that one. will update, thanks for spotting it